documents/validator.go

```diff
@@ -425,23 +416,14 @@ func attributeValidator(anchorSrv anchors.Service, idSrv identity.Service) Valid
 			}
 
 			signed := attr.Value.Signed
-			var aid anchors.AnchorID
-			aid, err = anchors.ToAnchorID(signed.DocumentVersion)
-			if err != nil {
-				return err
-			}
 
-			_, ats, erro := anchorSrv.GetAnchorData(aid)
-			if erro != nil {
-				// the attribute was added in this update itself.
-				// pick the update time from the model itself
-				ats = ts
-			}
+			payload := attributeSignaturePayload(signed.Identity.ToBytes(), model.ID(), signed.DocumentVersion, signed.Value)
 
-			payload := attributeSignaturePayload(signed.Identity[:], model.ID(), signed.DocumentVersion, signed.Value)
-			erri := idSrv.ValidateSignature(signed.Identity, signed.PublicKey, signed.Signature, payload, ats)
```

Comment:
I think I'm a bit confused by what you are suggesting since I already check the validity of the key in `identityService.ValidateKey()` by:
- getting the `revoked_at` block number, if any.
- checking if the latest block number >= revoked block number.

Is there some other check that should involve the model timestamp as well?